src/components/InputComponents/InputSearchAddressName.js

```diff
@@ -1,112 +1,61 @@
 import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-import _ from 'lodash';
+import Autocomplete from 'antd/es/auto-complete';
+import Spin from 'antd/es/spin';
 import cx from 'classnames';
 import { getUsersIdentityData } from '../../api/explorer';
-import TextInput from './TextInput';
 import styles from './styles.module.scss';
 
-function InputSearch({
-  id,
-  errorTitle,
-  register,
-  name,
-  placeholder,
-  validate,
-  setValue,
-  isRequired,
-  trigger,
-  defaultValue,
-}) {
-  const inputProps = {
-    errorTitle,
-    register,
-    name,
-    placeholder,
-    isRequired,
-    validate,
-    id,
-  };
-  const [inputValue, setInputValue] = useState(defaultValue || '');
+function InputSearch(props) {
+  const [loading, setLoading] = useState();
+  const [error, setError] = useState();
   const [suggestions, setSuggestions] = useState([]);
-  const [isSuggestedListShown, setIsSuggestedListShown] = useState(true);
```

Comment:
why refactor identity autocomplete?

Reply:
The component was opinionated towards RHF. Added back the obsolete component back which I will slowly eradicate through refactoring using ANTD